configs/build_config.py

```diff
@@ -97,7 +98,7 @@ def __init__(self, flask_config_path, autoratify=True):
         self._settings.update({
             'MANIFESTING_ROOT':     mroot,
             'FILESYSTEM_IMAGES':    fsimages,
-            'GOLDEN_IMAGE':         fsimages + '/golden/golden.arm.tar',
+            'GOLDEN_IMAGE':         fsimages + '/golden/golden.tar',
```

Comment:
Isn't this backward?  Don't you want the name of the tarball to have the arch in it?

Reply:
this is still "work in progress"

Reply:
Ok, maybe you should make a little more progress and then I can look at it :-)